src/main/java/com/j2k/worldbox/entity/model/CivilianModel.java

```diff
@@ -0,0 +1,62 @@
+package com.j2k.worldbox.entity.model;
+
+import com.j2k.worldbox.entity.custom.CivilianEntity;
+import com.mojang.blaze3d.matrix.MatrixStack;
+import com.mojang.blaze3d.vertex.IVertexBuilder;
+import net.minecraft.client.renderer.entity.model.BipedModel;
+import net.minecraft.client.renderer.model.ModelRenderer;
+
+public class CivilianModel<T extends CivilianEntity> extends BipedModel<T> {
+
+	protected CivilianModel(float modelSize, float yOffsetIn, int textureWidthIn, int textureHeightIn) {
+		super(modelSize, yOffsetIn, textureWidthIn, textureHeightIn);
+		// NOTE:XXX: There may be problems with rotation points
```

Comment:
can there still be problems here?

Reply:
I didn't force them to walk, so most likely there may be problems

Reply:
okey, thx